import-scripts/kill-import-hgnc-data.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
```

Comment:
cloned code? should we write something that can adjust based on which process is being managed

Reply:
yes ... on the next go around (genie) we should have shared functions